src/templates/generated/action.py.njx

```diff
@@ -0,0 +1,156 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+{% for pylint in data.pylints %}
+{{pylint}}
+{% endfor %}
+
+import argparse
+from collections import defaultdict
+from knack.util import CLIError
+
+
+class AddFactoryVstsConfiguration(argparse.Action):
```

Comment:
are these generated code of RPs? why they are in the template file?

Reply:
```
HISTORY.rst.njx
setup.cfg.njx
setup.py.njx
azext_metadata.json.njx
tests/__init__.py
tests/latest/preparers.py.njx
```
currently only these templates are in use. I just put the the action.py.njx and custom.py.njx and others in here for future usage. As these are very typical scenarios, we need to take them into consideration while designing the templates. You may ignore them for now.